gettingStarted.md

```diff
@@ -0,0 +1,354 @@
+    Legend({ displayComponent: true, layoutTag: NineLayoutTag.BOTTOM_RIGHT }),
+    Scale(merge({}, defaultOptions, { layoutTag: NineLayoutTag.BOTTOM_RIGHT })),
+    Toast(merge({}, defaultOptions, { layoutTag: NineLayoutTag.BOTTOM_MIDDLE })),
+  ]);
+};
+```
+
+### 5.3 Integrating the Plugins into the Application via polar-client.ts
+The `polar-client.ts` file ties everything together. It initializes the POLAR `core` instance, invokes the `addPlugins` function, and sets up the map with the defined configuration from `mapConfiguration.ts`.
+
+```Typescript
+import polarCore from '@polar/core';
+import { addPlugins } from './addPlugins';
+import { mapConfiguration } from './mapConfiguration';
+
+addPlugins(polarCore);
+
+const createMap = (layerConf) => {
+  polarCore.createMap({
+    containerId: 'polarstern',
+    mapConfiguration: {
+      ...mapConfiguration,
+      layerConf,
+    },
+  }).then(addStoreSubscriptions(/* subscription logic */));
+};
+
+// Initializes the layer list and creates the map instance
+polarCore.rawLayerList.initializeLayerList('https://geodienste.hamburg.de/services-internet.json', createMap);
+```
+
+### 5.4 Explanation of the Plugin Integration Process
+The process is as follows:
+
+  - `mapConfiguration.ts` defines the `core` settings and appearance of the map.
+  - `addPlugins.ts` handles the instantiation and configuration of individual plugins, utilizing `addPlugins(core)` to add them to the `core` instance.
+  - `polar-client.ts` combines these configurations to initialize and render the map, ensuring the plugins are integrated and functioning within the defined layout.
+
+### 5.5 Best Practices for Integrating Plugins
+- Centralized Configuration: Manage the primary map settings in mapConfiguration.ts for easy maintenance.
+- Modular Plugin Management: Use addPlugins.ts to separate plugin configurations, allowing for easier updates and modifications.
+- Testing: Regularly test your integrated plugins in the Snowbox environment to ensure compatibility and functionality.
+
+## 6. Using the Layout – The `NineLayout` System
+
+The `NineLayout` is a flexible layout system that allows plugins to be placed in a 3x3 grid (nine sections) on the map.
+
+### 6.1 How `NineLayout.vue` Works
+
+`NineLayout.vue` is the Vue component that represents the layout of the map and places plugins according to their configured positions.
+
+```vue
+<template>
+  <div class="wrapper">
+    <div
+      v-for="(tag, index) of tags"
+      :key="index"
+      :class="{
+        [tag]: true,
+        'has-window-size': hasWindowSize,
+      }"
+    >
+      <plugin-vessel
+        v-for="(pluginContainer, innerIndex) of getSortedTo(tag)"
+        :key="`${index}-${innerIndex}`"
+        v-bind="{ pluginContainer }"
+      />
+    </div>
+  </div>
+</template>
+```
+
+- The `wrapper` div is the container element for the entire layout.
+- Each plugin is positioned within the `plugin-vessel` component based on its `layoutTag`.
```

Comment:
I think this is an implementation detail that doesn't matter in this kind of document. I'd remove the whole chapter 6 and briefly explain layouting in the chapter "Configuration of the `iconMenu` Plugin", where the `NineLayoutTag` appears first. Explaining in short which options are available and that contents are pushed to a 3x3 grid seems sufficient to me.

Reply:
done

Reply:
The (previous) chapter 6 (now 5) does still exist

Reply:
It seems like I reintegrated this part while working on the file 😆 Now the layouting is explained briefly at the end of section 4.3.